src/components/ExerciseInfoComponent/ExerciseInfoPage.js

```diff
@@ -0,0 +1,87 @@
+import {React, useState} from 'react';
+import Bar from '../BarComponent/Bar';
+
+function ExerciseInfoPage() {
+
+  const buttonStyle = {
+    margin: '1vw',
+    borderRadius: '40px',
+    fontSize: '3vw',
+    fontFamily: 'Anonymous Pro',
+    background: '#ADADAD',
+    border: 'none',
+    textAlign: 'left',
+    float: 'right'
+  }
+
+  const inputStyle = {
+    borderRadius: '40px',
+    height: '70%',
+    width: '15%',
+    margin: '1vh',
+    fontFamily: 'Anonymous Pro',
+    fontSize: '3vw',
+    background: '#ADADAD',
+    border: 'none',
+  }
+
+  return (
+    <div>
+      <Bar />
+      <div style={{width: "100%", height: "80vh", alignItems: 'center', display:'flex', flexDirection: 'row'}}>
+        <div style={{width: "50%", height:'70%', backgroundColor: '#FFF8EB', border: "10px solid white", display:'flex', flexDirection: 'column', alignItems: 'center'}}>
+          <div style={{height: '10%', textAlign:'center', fontSize: '5vh', fontFamily: 'Anonymous Pro', fontWeight: 'bold', padding: '1vw'}}>
+            <div>
+                Bench Press
+            </div>
+          </div>
+
+          <div style={{padding:'1vw', width: '100%', height: '80%', textAlign: 'center', fontSize: '3vw', fontFamily: 'Anonymous Pro', alignItems: 'center', display:'flex', flexDirection: 'column', justifyContent: 'space-around'}}>
+            <iframe width="90%" height="90%" src="https://www.youtube.com/embed/-J4P-XOEtFY?si=tGVyWprfhzygOgfY"></iframe>
+          </div>
+        </div>
+        
+        <div style={{width: "50%", height:'70%', backgroundColor: '#FFF8EB', border: "10px solid white", display:'flex', flexDirection: 'column', alignItems: 'center'}}>
+          <div style={{height: '10%', fontSize: '4vh', fontFamily: 'Anonymous Pro', fontWeight: 'bold', padding: '1vw'}}>
+            <div style={{textAlign: 'center'}}>
```

Comment:
delete instruction section because they you need to write instructions for every single workout in the discover page